controllers/manager/manager.go

```diff
@@ -197,6 +198,12 @@ type PlatformManager struct {
 	GetPlatformClientImpl           func(namespace string) *gophercloud.ServiceClient
 }
 
+type RestoreStatus struct {
```

Comment:
It's weird to have the RestoreStatus in the manager, but only designed for platform networks, can you rename it if you want to use it?

Reply:
RestoreStatus is not only designed for platform network.  It is common across all the controllers. It will be used by other controllers like ptp instance, ptp interface and system controller,